app/components/header.tsx

```diff
@@ -16,12 +16,11 @@ import {
 import { TriangleUpIcon, LockIcon, HamburgerIcon } from '@chakra-ui/icons';
 import { Link, useFetcher } from '@remix-run/react';
 
-import { useUser } from '~/utils';
+import { useOptionalUser } from '~/utils';
 
 export default function Header() {
-  const user = useUser();
+  const user = useOptionalUser();
```

Comment:
@Ririio does this change cause any issues elsewhere in our code?

Reply:
I don't think it will, the purpose was merely to display the username, it might affect the testing, because we require a rendering of the username in that specific area, but I don't think it will cause any problems